node/sc/subbridge.go

```diff
@@ -173,9 +173,10 @@ func NewSubBridge(ctx *node.ServiceContext, config *SCConfig) (*SubBridge, error
 		ctx:            ctx,
 		chainHeadCh:    make(chan blockchain.ChainHeadEvent, chainHeadChanSize),
 		logsCh:         make(chan []*types.Log, chainLogChanSize),
-		//txCh:            make(chan blockchain.NewTxsEvent, transactionChanSize),
-		requestEventCh: make(chan RequestValueTransferEvent, tokenReceivedChanSize),
-		handleEventCh:  make(chan HandleValueTransferEvent, tokenTransferChanSize),
+		//txCh:            make(chan blockchain.NewTxsEvent,
```

Comment:
How about removing the comment?

Reply:
The comment is not related with this PR. :)
I will consider remove it in another PR.